File: Mod09/Labfiles/Solution/VB/File_IO_VB/File_IO_VB/Instructor.vb
Public Class Instructor
    Inherits Person

    Public Sub New(ByVal first As String, ByVal last As String, ByVal gender As String, ByVal age As Integer)

        Me.FirstName = first
        Me.LastName = last
        Me.Gender = gender
        Me.Age = age

    End Sub
    Public Overrides Sub move()

    End Sub
End Class

File: Mod09/Labfiles/Solution/CS/File_IO_CS/File_IO_CS/Instructor.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace File_IO_CS
{
    class Instructor : Person
    {
        public Instructor(string first, string last, string gender, int age)
        {
            this.FirstName = first;
            this.LastName = last;
            this.Gender = gender;
            this.Age = age;
        }

        public override void move()
        {

        }
    }
}
